--- a/cmake/FindQScintilla.cmake
+++ b/cmake/FindQScintilla.cmake
@@ -4,7 +4,7 @@
 #  QScintilla_LIBRARIES - libraries needed
 
 FIND_PATH(QSCINTILLA_INCLUDE_DIR NAMES Qsci/qsciscintilla.h PATH_SUFFIXES qt5 qt)
-FIND_LIBRARY(QSCINTILLA_LIBRARY NAMES qsciscintilla qt5scintilla2 qscintilla2)
+FIND_LIBRARY(QSCINTILLA_LIBRARY NAMES qsciscintilla qt5scintilla2 qscintilla2 qscintilla2_qt5)
 
 SET(QScintilla_INCLUDE_DIRS ${QSCINTILLA_INCLUDE_DIR})
 SET(QScintilla_LIBRARIES ${QSCINTILLA_LIBRARY})
